<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>大学生时间管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/C:/Users/13677/Downloads/20250506_2335_学生管理系统图标_simple_compose_01jtk3rb2reex8ecbp5vwkpme8.png</normaloff>:/C:/Users/13677/Downloads/20250506_2335_学生管理系统图标_simple_compose_01jtk3rb2reex8ecbp5vwkpme8.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 12pt;
    font-weight: normal;
background-color: rgb(223, 230, 233);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #f3f3f3;
    color: black;
    border: none;
      border-radius: 10px;
     padding: 8px 16px;
      font-size: 14px;
    }
    QPushButton:hover {
     background-color: #bababa;
    }
    QPushButton:pressed {
      background-color: #858585;
    }</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>TO-DO</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="delTaskButton">
       <property name="text">
        <string>点击删除</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="delTaskButton_2">
       <property name="text">
        <string>查询删除</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="srcTaskButton">
       <property name="text">
        <string>查询事务</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="srcChgButton">
       <property name="text">
        <string>查询修改</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="addTaskButton">
       <property name="text">
        <string>添加事务</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="tdlistButton">
       <property name="text">
        <string>TDLIst</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="chgTaskButton">
       <property name="text">
        <string>点击修改</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="concentrateBtn">
       <property name="text">
        <string>专注模式</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font-size:10px;</string>
       </property>
       <property name="text">
        <string>双击事件
添加提醒</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>469</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
/* 整体外观 */
QCalendarWidget {
    background-color: #fefefe;
    border: 10px solid #8e8e8e;
    border-radius: 10px;
}

/* 顶部导航栏背景 */
QWidget#qt_calendar_navigationbar {
    background-color: #2c3e50;
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
}

/* 年月切换按钮 */
QCalendarWidget QToolButton {
    font-size: 18px;
    color: #ffffff;
    background-color:#2c3e50;
    border-radius: 6px;
    padding: 5px;
}
QCalendarWidget QAbstractItemView {
    font-size: 16px;
    selection-background-color: #6BBE92;
}


QCalendarWidget QToolButton:hover {
    background-color: #2980b9;
}

/* 下拉菜单（选择月份和年份） */
QCalendarWidget QMenu {
    background-color: white;
    border: 1px solid #bbb;
    padding: 5px;
}

 日期表格字体与颜色 
QCalendarWidget QAbstractItemView {
    selection-background-color: #3498db;
    selection-color: white;
    font-size: 14px;
    gridline-color: #e0e0e0;
}

/* 鼠标悬停效果 */
QCalendarWidget QAbstractItemView::item:hover {
    background-color: #ecf0f1;
}

/* 无效日期（上月/下月的灰色数字） */
QCalendarWidget QAbstractItemView::item:disabled {
    color: #bdc3c7;
}

/* 当前选中的日期 */
QCalendarWidget QAbstractItemView::item:selected {
    background-color: #3498db;
    color: white;
    border-radius: 4px;
}

/* 今日高亮（通过 QTextCharFormat 更精细控制） *</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>469</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
/* 整体边框与背景 */
QTableWidget {
    background-color:#ffffff;
    border: 5px solid #dcdcdc;
    gridline-color: #e0e0e0;
    font-size: 14px;
    selection-background-color: #3498db;
    selection-color: #ffffff;
}

/* 表头（horizontalHeader） */
QHeaderView::section {
    background-color: #2c3e50;
    color: white;
    padding: 6px;
    border: none;
    font-weight: bold;
}

/* 鼠标悬停单元格（无选中时） */
QTableWidget::item:hover {
    background-color: #f1f1f1;
}

/* 单元格选中样式 */
QTableWidget::item:selected {
    background-color: #3498db;
    color: white;
}

/* 去掉焦点虚线边框 */
QTableWidget::focus {
    outline: none;
}

/* 滚动条背景 */
QScrollBar:vertical {
    background: #f0f0f0;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

/* 滚动条滑块 */
QScrollBar::handle:vertical {
    background: #b1b1b1;
    min-height: 20px;
    border-radius: 5px;
}

/* 滚动条滑块悬停效果 */
QScrollBar::handle:vertical:hover {
    background: #2980b9;
}

/* 上箭头 */
QScrollBar::sub-line:vertical {
    height: 0px;
}

/* 下箭头 */
QScrollBar::add-line:vertical {
    height: 0px;
}

/* 剩余区域 */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

/* 横向滚动条同理 */
QScrollBar:horizontal {
    background: #f0f0f0;
    height: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal {
    background: #b1b1b1;
    min-width: 20px;
    border-radius: 5px;
}

QScrollBar::handle:horizontal:hover {
    background: #2980b9;
}

QScrollBar::sub-line:horizontal,
QScrollBar::add-line:horizontal {
    width: 0px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
    </property>
    <property name="rowCount">
     <number>20</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>事务编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>事务标题</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>开始时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>结束时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>优先级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>事务状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>事务描述</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
